.env.deploy.vm.example

```
@@ -0,0 +1,38 @@
+PLACE_SEARCH_BOOTSTRAP_ENABLED=true
+PLACE_SEARCH_BOOTSTRAP_BATCH_SIZE=500
+PLACE_SEARCH_BOOTSTRAP_LOCK_ID=91000001
+
+S3_ENDPOINT=http://minio:9000
+S3_REGION=us-east-1
+S3_ACCESS_KEY=change-me
+S3_SECRET_KEY=change-me
+S3_BUCKET=insideout-floorplan
+
+SECURITY_JWT_SECRET=replace-with-long-random-secret
+SECURITY_JWT_ACCESS_TOKEN_EXPIRATION_MS=3600000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SECURITY_JWT_ACCESS_TOKEN_EXPIRATION_MS key should go before the SECURITY_JWT_SECRET key

(UnorderedKey)